packages/main/src/Carousel.ts

```diff
@@ -549,19 +545,37 @@ class Carousel extends UI5Element {
 	}
 
 	get effectiveItemsPerPage(): number {
+		const itemsPerPageArray = this.itemsPerPage.split(" ");
+
+		itemsPerPageArray.forEach(element => {
```

Comment:
like this the items are not reset to default value 1 in case when the string does not contain e.g. M or L. Make them reset to default value when a size is skipped